File: server/java/spring/content/src/main/java/xyz/hoper/content/entity/Content.java
package xyz.hoper.content.entity;


import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;


/**
 * @Description TODO
 * @Date 2022/11/21 15:18
 * @Created by lbyi
 */
@Table(name="content",schema="public")
@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Content implements Serializable {
    @Id
    Long id;

    String name;
}

File: server/java/spring/content/src/main/kotlin/xyz/hoper/content/entity/Content.kt
package xyz.hoper.content.entity


import jakarta.persistence.*
import kotlinx.serialization.Serializable



/**
 * @Description TODO
 * @Date 2022/11/21 15:18
 * @Created by lbyi
 */
@Table(name="content",schema="public")
@Serializable
@Entity
class Content {
    @Id
    var id: Long = 0

    var name:String = ""
}